lib/where-or.rb

```diff
@@ -1,6 +1,6 @@
 abort "Congrats for being on Rails 5. Now please remove this patch by getting rid of the `where-or` gem" if ActiveRecord::VERSION::MAJOR > 4
 # Tested on Rails Rails 4.2.3
-warn "Patching ActiveRecord::Relation#or.  This might blow up" if ActiveRecord.version.to_s < '4.2.3'
+warn "Patching ActiveRecord::Relation#or.  This might blow up" if ActiveRecord.version < Gem::Version.new('4.2.3')
```

Comment:
To be fair, I wrote this warning when this was a monkey patch in my app and I *wanted* it to warn whenever we upgraded rails so I could confirm it worked and then bump the number. 

Changing it to 4.2.10 will make it warn on 4.2.8 which isn't desirable, I don't think.

I'd be okay removing this line altogether. I'm not sure it adds much to the usage of the code in a gem.

Should probably `require 'rubygems/version'` first, though, if we're going to change it.

Reply:
only people who are using rails < 4.2.3 will get the warning

Reply:
yes, so maybe we will using #15

Reply:
@Eric-Guo No problem, but I don't know about advantage of doing that explicitly over a convenient wrapper  `Gem::Version`
Also since `ActiveRecord.version` is already an instance of the class `Gem::Version` we don't have to load it explicitly, it is already available

Reply:
I think I meant that I have no idea if it works on < 4.2.3 since that's probably what I implemented it on